Code verification tutorial for TMS (#18281)
idaholab/moose: modules/doc/content/getting_started/examples_and_tutorials/tutorial03_verification/step04_mms.md

# Method of Manufactured Solution

## Introduction

> In practice an exact solution to the problem being solved is not known, since there would

Reviewer: ```suggestion
In practice an exact solution to the problem being solved is rarely known, since there would
```

be no reason to perform a simulation if the answer is known. If the solution

> is not known then the error can not be known. However, it is possible to "manufacture" a solution

Reviewer: ```suggestion
is not known then the error cannot be computed. However, it is possible to "manufacture" a solution
```

> using a method known as [!ac](MMS). In this step of the tutorial the two-dimensional heat equation

Reviewer: ```suggestion
using a method known as the [!ac](MMS). In this step of the tutorial the two-dimensional heat equation
```
